src/elements/content-sidebar/SidebarNav.scss

```diff
@@ -63,3 +63,88 @@
         }
     }
 }
+
+.is-responsive-web {
+    @include breakpoint($small-screen) {
+        .bcs .bcs-NavButton,
+        .bcs-SidebarNav-secondary .btn-plain,
+        .bdl-AdditionalTab {
+            width: 46px;
```

Comment:
Why did we lose a couple pixels here?

Reply:
It looks like I asked this and it was answered [previously](https://github.com/box/box-ui-elements/pull/2917#discussion_r907874219). Can we add a brief comment in the code to avoid confusion in the future?

Reply:
Added a comment to clarify.